pkg/checkmarxone/reporting.go

```diff
@@ -0,0 +1,285 @@
+		{Description: "Medium to verify issues", Details: fmt.Sprint((*data)["Medium"].(map[string]int)["ToVerify"])},
+		{Description: "Low issues", Details: fmt.Sprint((*data)["Low"].(map[string]int)["Issues"])},
+		{Description: "Low not false positive issues", Details: fmt.Sprint((*data)["Low"].(map[string]int)["NotFalsePositive"])},
+		{Description: "Low not exploitable issues", Details: fmt.Sprint((*data)["Low"].(map[string]int)["NotExploitable"])},
+		{Description: "Low confirmed issues", Details: fmt.Sprint((*data)["Low"].(map[string]int)["Confirmed"])},
+		{Description: "Low urgent issues", Details: fmt.Sprint((*data)["Low"].(map[string]int)["Urgent"])},
+		{Description: "Low proposed not exploitable issues", Details: fmt.Sprint((*data)["Low"].(map[string]int)["ProposedNotExploitable"])},
+		{Description: "Low to verify issues", Details: fmt.Sprint((*data)["Low"].(map[string]int)["ToVerify"])},
+		{Description: "Informational issues", Details: fmt.Sprint((*data)["Information"].(map[string]int)["Issues"])},
+		{Description: "Informational not false positive issues", Details: fmt.Sprint((*data)["Information"].(map[string]int)["NotFalsePositive"])},
+		{Description: "Informational not exploitable issues", Details: fmt.Sprint((*data)["Information"].(map[string]int)["NotExploitable"])},
+		{Description: "Informational confirmed issues", Details: fmt.Sprint((*data)["Information"].(map[string]int)["Confirmed"])},
+		{Description: "Informational urgent issues", Details: fmt.Sprint((*data)["Information"].(map[string]int)["Urgent"])},
+		{Description: "Informational proposed not exploitable issues", Details: fmt.Sprint((*data)["Information"].(map[string]int)["ProposedNotExploitable"])},
+		{Description: "Informational to verify issues", Details: fmt.Sprint((*data)["Information"].(map[string]int)["ToVerify"])},
+	}
+	for _, detailRow := range detailRows {
+		row := reporting.ScanRow{}
+		row.AddColumn(detailRow.Description, 0)
+		row.AddColumn(detailRow.Details, 0)
+
+		detailTable.Rows = append(detailTable.Rows, row)
+	}
+	scanReport.DetailTable = detailTable
+
+	return scanReport
+}
+
+func CreateJSONReport(data *map[string]interface{}) CheckmarxOneReportData {
+	checkmarxReportData := CheckmarxOneReportData{
+		ToolName:         `checkmarxone`,
+		ProjectName:      fmt.Sprint((*data)["ProjectName"]),
+		GroupName:        fmt.Sprint((*data)["Group"]),
+		GroupPath:        fmt.Sprint((*data)["GroupFullPathOnReportDate"]),
+		DeepLink:         fmt.Sprint((*data)["DeepLink"]),
+		Preset:           fmt.Sprint((*data)["Preset"]),
+		CheckmarxVersion: fmt.Sprint((*data)["CheckmarxVersion"]),
+		ScanType:         fmt.Sprint((*data)["ScanType"]),
+	}
+
+	if s, err := strconv.ParseInt(fmt.Sprint((*data)["ProjectId"]), 10, 64); err == nil {
+		checkmarxReportData.ProjectID = s
+	}
+
+	if s, err := strconv.ParseInt(fmt.Sprint((*data)["ScanId"]), 10, 64); err == nil {
```

Comment:
No longer need to convert Int to Str since ScanID in Cx1 is already a string
```suggestion
	ScanID: fmt.Sprint((*data)["ScanID"])
```

Reply:
Done